packages/components/src/beta-badge/BetaBadge.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { __ } from "@wordpress/i18n";
+
+/**
+ * Function for the BetaBadge component.
+ *
+ * @param {bool} inLabel Whether the BetaBadge is displayed close to a Label.
+ *
+ * @returns {React.Component} The BetaBadge.
+ */
+const BetaBadge = ( { inLabel } ) => (
```

Comment:
The beta badge still works without the `inLabel` property, so I would remove it from this component.